### ISSUE TITLE
[WEB] [Employer-Profiles Search] Incorrect Behaviour of Profile View Counts

### ISSUE THREAD

**Issue by eraytech** [gak112/DearJobTesting#946]
**Steps:**
1. Open a web browser and navigate to the website: "https://dearjob.org".
2. On the website's homepage, locate and click on the "LOGIN" link in the menu tab.
3. Enter the registered Employer email ID and password, then click on the login button or press enter.
4. After successful login, ensure that the profile has an active subscription plan. Click on "Search" on the home page.
5. In the Profiles Search page, review the list of candidates.
6. Observe the right-side section of the page displaying current plan details, including Profile Views, Viewed Profiles, and Balance Profiles.
7. Allow the subscription to expire or manually end it.

**Actual Result:**
- After the subscription ends, the counts for Profile Views, Viewed Profiles, and Balance Profiles continue displaying the previous count without resetting, even though the subscription has ended.

**Expected Result:**
- Once the subscription ends, the Profile Views, Viewed Profiles, and Balance Profiles counts should reset to zero or adjust according to the limitations of the new plan or lack of a subscription. These counts should reflect the allowances specified by the new or expired plan. They should not persist or continue from their previous count but should reset to comply with the current subscription privileges or no subscription.

![image](https://github.com/gak112/DearJobTesting/assets/101229315/ac0426ec-f9e7-4628-bb2e-ccd52b93befc)

**Comment by eraytech** [comment]
will be using this id : kusumamounicatester+employer@gmail.com, to check whether everthing gets reset once subscription is done, video attached for refernce of id before changes


https://github.com/gak112/DearJobTesting/assets/101229315/aa6478f4-d508-4c16-b3a1-5a2fc6ccdee2

kusumamounicatester+51@gmail.com

kusumamounicatester+03@gmail.com

**Comment by eraytech** [comment]
verified and working
